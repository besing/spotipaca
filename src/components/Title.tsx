import React from 'react';
import styled from 'styled-components';
import { COLORS } from '../utils/constants';

const StyledTitle = styled.h1`
  font-family: 'Abril Fatface', 'Roboto', sans-serif;
  color: ${COLORS.spotifyGreen};
  font-weight: normal;
  font-size: 50px;
  text-align: center;
  margin: 70px 20px;
`;

const Title = () => <StyledTitle>spotipaca</StyledTitle>;

export default Title;
