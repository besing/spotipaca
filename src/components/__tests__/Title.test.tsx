import React from 'react';
import { cleanup, render } from '@testing-library/react';
import 'jest-styled-components';
import '@testing-library/jest-dom/extend-expect';

import Title from '../Title';

afterEach(cleanup);

describe('<Title />', () => {
  it('renders the component', () => {
    const { container } = render(<Title />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
