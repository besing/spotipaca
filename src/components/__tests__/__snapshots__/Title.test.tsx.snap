// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders the component 1`] = `
.c0 {
  font-family: 'Abril Fatface','Roboto',sans-serif;
  color: #1db954;
  font-weight: normal;
  font-size: 50px;
  text-align: center;
  margin: 70px 20px;
}

<h1
  class="c0"
>
  spotipaca
</h1>
`;
